src/client/Microsoft.Identity.Client/ApiConfig/Executors/ConfidentialClientExecutor.cs

```diff
@@ -91,7 +94,7 @@ public ConfidentialClientExecutor(IServiceBundle serviceBundle, ConfidentialClie
         public async Task<Uri> ExecuteAsync(
             AcquireTokenCommonParameters commonParameters,
             GetAuthorizationRequestUrlParameters authorizationRequestUrlParameters,
-            CancellationToken cancellationToken)
+            CancellationToken cancellationToken) // TODO: propagate cancellation token
         {
```

Comment:
will the cancellation token todos be addressed in this PR? if not, can we add a note to the right work item? maybe this one [https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1582](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/1582) or create another one.

Reply:
Can do, just spotted that the param was not used and added a comment.